src/components/atoms/Input/Input.tsx

```diff
@@ -12,17 +13,20 @@ const Input = ({
   handleOnChange,
   ...props
 }: IProps): JSX.Element => {
-  const renderLabel = () => label && <Label>{label}</Label>;
+  const [inputId] = useState(newId('input-'));
```

Comment:
We don't need useState here to achieve the same results

```suggestion
  const inputId = newId('input-');
```